### ISSUE TITLE
Y for confirmation 

### ISSUE THREAD

**Issue by VladKhomich** [VladKhomich/jiralog#11]
Based on user feedback, it would be nice to add and option to use Y and N for confirmation and declining

**Comment by prihodkoanton** [comment]
Yes, it would be great if it also worked with "y" and "n" (lowercase letters)

**Comment by VladKhomich** [comment]
added in v0.1.6
